Include/PGE/Graphics/Texture.h

```diff
@@ -48,6 +48,7 @@ class Texture : private PolymorphicHeap {
                 : width(width), height(height), buffer(buffer), size(size) { }
             Mipmap() = default;
         };
+        // TODO: Can be replaced with Enumerable with C++20 Modules
```

Comment:
Can't we include the Concepts header here?

Reply:
We can include the Concepts header, but we can't move the function definition to the header since it relies on a Graphics implementation, creating a cyclic dependancy. (Functions using a concept auto as a parameter are basically shorthand templates.)